lib/QubitFlatfileImport.class.php

```diff
@@ -638,6 +642,22 @@ public function row($row = array())
             )
           );
 
+          // Unable to match to keymap entry.
+          // Try matching via secondary method - match record in DB by
+          // identifier + title + repository.
+          if ($this->columnExists('identifier')
```

Comment:
Feel free to do a few conditions on a single line as long as it's <=120 characters.